R_program/CallSTEPWAT2.R

```diff
@@ -170,13 +252,11 @@ foreach (g = 1:length(GCM)) %dopar% { # loop through all the GCMs
             #use with Vic weather database and all new weather databases
             if(database_name!="dbWeatherData_Sagebrush_KP.v3.2.0.sqlite")
             {
-              downscaling_method <- "hybrid-delta-3mod"
-              setwd(paste("Site_",s,"_hybrid-delta-3mod.",y,".",r,".",GCM[g], sep=""))
-              weath.read<-paste(assembly_output,"Site_",s,"_hybrid-delta-3mod.",y,".",r,".",GCM[g], sep="")
+              weather.dir <- paste("Site_",s,"_hybrid-delta-3mod.",y,".",r,".",GCM[g], sep="")
```

Comment:
I propose to change this to weather.read.dir, so it does not get confused with weather.dir below.